### ISSUE TITLE
Работа над ошибками

### ISSUE THREAD

**Issue by Alecto** [Julia45/Skill-up#1]
1)
есть отличия во внешнем виде логотипа, проверьте попиксельным соответствием либо сверьте с мастер0\-шаблоном
![image](https://user-images.githubusercontent.com/8464662/82083787-f3843000-96f2-11ea-89c5-78ce39de0f7f.png)

2)
3)
контейнеры отстутсвуют
![image](https://user-images.githubusercontent.com/8464662/82083976-3cd47f80-96f3-11ea-97b8-6ce8c2a46da7.png)

4)
аналогично
![image](https://user-images.githubusercontent.com/8464662/82084025-4fe74f80-96f3-11ea-9552-aa02e5ef6a6d.png)

5)
цвет не соответствует
6)
оутлайна быть не должно
![image](https://user-images.githubusercontent.com/8464662/82084086-6ee5e180-96f3-11ea-9f61-09f54bd4bb0b.png)

**Comment by Julia45** [comment]
а вот же мои контейнеры 
https://prnt.sc/shtom8
и вот: https://prnt.sc/shtoy6
Или должны быть какие-то еще?)

**Comment by Alecto** [comment]
они не работают и не ограничивают контент.

сб, 16 трав. 2020 о 11:54 Juliya Mankovskaya <notifications@github.com>
пише:

**Comment by Julia45** [comment]
Здравствуйте! Исправила ошибки, осталась только одна непонятка с контейнерами. Перепроверила, исправила div на section в секции about. Cейчас выглядит как у вас. Посмотрите пожалуйста

**Comment by Alecto** [comment]
ссылочки скините? из гитхаба приходит только текст

нд, 17 трав. 2020 о 00:45 Juliya Mankovskaya <notifications@github.com>
пише:

**Comment by Julia45** [comment]
Да, конечно: https://github.com/Julia45/Skill-up/tree/master/Grid-template и https://julia45.github.io/Skill-up/Grid-template/

**Comment by Julia45** [comment]
Отправила)

Я еще хотела вас предупредить, по семейный обстоятельствам, я не смогу
сегодня присутствовать на лекции:(

Я обязательно ее отработаю и пришлю дз))

**Comment by Alecto** [comment]
есть запись,
посмотрите, главное - чтобы разобрались )

нд, 17 трав. 2020 о 11:16 Juliya Mankovskaya <notifications@github.com>
пише:

> Отправила)
>
> Я еще хотела вас предупредить, по семейный обстоятельствам, я не смогу
> сегодня присутствовать на лекции:(
>
> Я обязательно ее отработаю и пришлю дз))
>

**Comment by Julia45** [comment]
ох, до меня дошло)) Посмотрите пожалуйста: https://julia45.github.io/Skill-up/Grid-template/
https://github.com/Julia45/Skill-up/tree/master/Grid-template

**Comment by Alecto** [comment]
5- цвет не исправили, так и остался серый.

**Comment by Alecto** [comment]
6)
адаптация не выполнилась
![image](https://user-images.githubusercontent.com/8464662/82255703-c38c9500-995d-11ea-87e8-7ac7b2ddebbb.png)

7)
тоже можно исправить
![image](https://user-images.githubusercontent.com/8464662/82255861-09495d80-995e-11ea-8a86-09e92384fd1e.png)

**Comment by Alecto** [comment]
осталось несколько некритичных замечаний.
посмотрите и исправьте их.

и таких строчек по хорошему быть не должно, приведите все к миксинам.
[image: image.png]

работа принята.

пн, 18 трав. 2020 о 15:49 Juliya Mankovskaya <notifications@github.com>
пише:
